spec/rspec/core/configuration_options_spec.rb

```diff
@@ -198,16 +210,24 @@
   end
 
   describe "--no-color" do
-    it "sets :color => false" do
-      expect(parse_options('--no-color')).to include(:color => false)
+    it "sets :no_color => true" do
+      expect(parse_options('--no-color')).to include(:no_color => true)
+    end
+
+    it "overrides previous :force_color => true" do
+      expect(parse_options('--force-color', '--no-color')).to include(:no_color => true)
+      expect(parse_options('--force-color', '--no-color')).to_not include(:force_color => true)
```

Comment:
This can be combined into one compound expectation:

``` ruby
expect(parse_options('--force-color', '--no-color')).to include(:no_color => true).and exclude(:force_color => true)
```